test/connectors/SubmissionConnectorSpec.scala

```diff
@@ -35,15 +36,17 @@ class SubmissionConnectorSpec extends Connector {
   lazy val connector: SubmissionConnector = app.injector.instanceOf[SubmissionConnector]
   val conversationId: ConversationId      = ConversationId("UUID")
   val submitUrl                           = "/mandatory-disclosure-rules/submit"
+  val submitxmlUrl                        = "/mandatory-disclosure-rules/submitxml"
```

Comment:
I don't think this is needed here now, other than that LGTM

Reply:
Line removed